### ISSUE TITLE
TaskFailed - taskid:hmmscan:longest_orfs.pep.x.Pfam-A.hmm

### ISSUE THREAD

**Issue by truelalin** [dib-lab/dammit#153]
Hi, 
I built a transcriptome using rnaSpades and am trying to annotate it.
Unfortunately, there was an error during the process that popped up:

Command failed: 'cat /home/lalini/igor_projects/transcripts.fasta.dammit/transcripts.fasta.transdecoder_dir/longest_orfs.pep | /home/lalini/anaconda3/envs/dammit/bin/parallel --round-robin --pipe -L 2 -N 10000 --gnu -j 40 -a /home/lalini/igor_projects/transcripts.fasta.dammit/transcripts.fasta.transdecoder_dir/longest_orfs.pep /home/lalini/anaconda3/envs/dammit/bin/hmmscan --cpu 1 --domtblout /dev/stdout -E 1e-05 -o /home/lalini/igor_projects/transcripts.fasta.dammit/transcripts.fasta.transdecoder_dir/longest_orfs.pep.x.pfam.tbl.hmmscan.out /home/lalini/.dammit//databases/Pfam-A.hmm /dev/stdin > /home/lalini/igor_projects/transcripts.fasta.dammit/transcripts.fasta.transdecoder_dir/longest_orfs.pep.x.pfam.tbl' returned 24

########################################
TaskFailed - taskid:hmmscan:longest_orfs.pep.x.Pfam-A.hmm
hmmscan:longest_orfs.pep.x.Pfam-A.hmm <stderr>:
parallel: Warning: A record was longer than 1048576. Increasing to --blocksize 1363150.
parallel: Warning: A record was longer than 1363150. Increasing to --blocksize 1772096.
parallel: Warning: A record was longer than 1772096. Increasing to --blocksize 2303726.
parallel: Warning: A record was longer than 2303726. Increasing to --blocksize 2994845.
parallel: Warning: A record was longer than 2994845. Increasing to --blocksize 3893300.

hmmscan:longest_orfs.pep.x.Pfam-A.hmm <stdout>:

I am a beginner in coding. Can anybody decipher this for me?

Thank you

**Comment by truelalin** [comment]
The station running it is 40 cores and I ran dammit at --n-threads 40. This is where I noticed the above problem first. Then, I went back removed sphinx installed by Anaconda and let dammit install it's own. Then tried running:
dammit annotate transcriptome.fasta --database /.dammit/databases/ --busco-group fungi --n_threads 20 --user-databases /home/igor_projects/ data/P.sojae.pep.fasta

It's worked overnight and still on now at the cmscan:transcripts.fasta.x.Rfam.cm

Hopefully, dropping the number of threads was the solution.
